service/catalog/model.go

```diff
@@ -960,7 +960,7 @@ type CreateMonitor struct {
 	// slices.
 	SlicingExprs []string `json:"slicing_exprs,omitempty"`
 	// Configuration for monitoring snapshot tables.
-	Snapshot *MonitorSnapshotProfileType `json:"snapshot,omitempty"`
+	Snapshot MonitorSnapshotProfileType `json:"snapshot,omitempty"`
```

Comment:
This pointer should not be removed. This was a bug which was fixed last week. I may have forgotten to update the SDK version in the deco tool.
Relevant PR
https://github.com/databricks/databricks-sdk-go/pull/834

Reply:
updated, will update the go sdk in deco to official release